Makefile

```diff
@@ -104,6 +105,17 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
+##@ Chart Testing
+
+.PHONY: ct-lint
+ct-lint: ## Lint and validate a chart.
```

Comment:
This rule looks to be not called by any functions. Are you going to update github actions? or it will be called by artifacthub?

Reply:
Running the ct-lint command in GitHub Actions seems to fail.
So I created the ct-lint command as a command to test Helm Chart locally.